### ISSUE TITLE
Enhance a11y for `FileInput`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5407]
name: FileInput

### Description
2 accessibility violation was found in the `FileInput`.

#### [Components / FileInput / Children](https://yamada-ui.github.io/yamada-ui/?path=/story/components-fileinput--children)

Received:
ARIA commands must have an accessible name (aria-command-name)

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-command-name?application=axeAPI

#### [Components / FileInput / ReactHookForm](https://yamada-ui.github.io/yamada-ui/?path=/story/components-fileinput--react-hook-form)

Received:
ARIA commands must have an accessible name (aria-command-name)

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/aria-command-name?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
